raven/processes/wps_forecast_evaluation.py

```diff
@@ -0,0 +1,154 @@
+import json
+
+from pywps import ComplexInput, ComplexOutput
+from pywps import FORMATS
+from pywps import LiteralInput
+from pywps import Process
+from pywps.app.Common import Metadata
+
+import numpy as np
+from scipy.stats import norm
+import xskillscore as xs
+import xarray as xr
+
+class ForecastEvaluationProcess(Process):
+    def __init__(self):
+        inputs = [ComplexInput('obs', 'Stream flow observation',
+                               abstract='Steam flow observation time series',
+                               supported_formats=(FORMATS.NETCDF,)),
+                  ComplexInput('fcst', 'Stream flow forecast',
+                               abstract='Stream flow forecast time series, deterministic or ensemble',
+                               supported_formats=(FORMATS.NETCDF,)),
+                  LiteralInput('skip_nans', 'Skip NaNs in metric evaluation',
+                               abstract='Skip NaNs in forecast evaluation computations',
+                               data_type='boolean',
+                               default = True,
+                               min_occurs=0,
+                               max_occurs=1),
+                  LiteralInput('BSS_threshold', 'Threshold for Brier Skill Score exceeding given threshold',
+                               abstract='Threshold for Brier Skill Score exceeding given threshold',
```

Comment:
Ideally the thresholds should remain as it's usual to check the skill for given values, and so that flexibility is nice. However I understand that the code is very... redundant this way. If it's possible to keep, I would. Would using climpred/hydrostats improve this?

Reply:
So the threshold_brier_score is still there. I removed crps_quadrature, crps_gaussian and 
brier_score.

Reply:
Sounds good to me!

So you basically converted my 100 lines into 3 lines... crazy stuff!

I think we're good!